src/vxworks/mod.rs

```diff
@@ -0,0 +1,2322 @@
+//! Hacking together the definitions for VxWorks Bindings
```

Comment:
Could you run `rustfmt` on this file ?

e.g. `rustfmt src/vxworks/mod.rs` ? 

We have temporarily disabled the formatting check on CI, but as `rustfmt` fixes some bugs we plan to enable it again.

Reply:
Sure.